tests/plugins/test_amp_plugins.py

```diff
@@ -45,6 +45,7 @@ class MyApexPlugin(ApexMixedPrecisionPlugin):
         "SLURM_LOCALID": "0",
     },
 )
+@mock.patch("torch.cuda.is_available", return_value=True)
```

Comment:
I wonder if we could get away with patching the newly introduced staticmethods instead of torch's. No need to do this in this PR